gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -15,14 +15,33 @@ security:
     - CookieAuth: []
 
 servers:
-    - url: /management/v2
-      description: Gravitee.io APIM - Management API - v2
-    - url: /management/v2/organizations/{orgId}
-      description: Gravitee.io APIM - Management API - v2
+    - url: "{protocol}://{gatewayHost}/management/v2"
+      description: APIM Management API v2 - Default base URL
       variables:
-        orgId:
-            default: DEFAULT
-            description: Allow to target a specific organization
+      protocol:
```

Comment:
I think there is an indentation issue here, it could explain the following error in CI:
<img width="1510" alt="image" src="https://github.com/gravitee-io/gravitee-api-management/assets/4112568/b38f80da-56ca-49fb-b089-60f30c647790">